python/py-eyed3/Portfile

```diff
@@ -56,8 +57,12 @@ if {${name} ne ${subport}} {
     } else {
     # list here dependencies for Python != 2.7
         depends_lib-append \
-                    port:py${python.version}-deprecation \
-                    port:py${python.version}-filetype
+            port:py${python.version}-coverage \
+            port:py${python.version}-deprecation \
+            port:py${python.version}-filetype \
+            port:py${python.version}-packaging \
+            port:py${python.version}-parsing \
+            port:py${python.version}-toml
```

Comment:
none of the dependencies you added are actually needed at buildtime/runtime... it's true that upstream has listed them in the `setup.py` under `install_requires` but that doesn't seem accurate. If you actually extract the package form PyPI (as we do for this port) and check the `setup.py` that comes with that you'll see that none of these are listed. It's likely some leftover (or requirement) for the switch to `poetry` which they likely use to make the distribution.

Reply:
i will double check every dependences. before working on the portfile, I build the package in virtualenv and you are correct saying pip list requirement reports the whole list I added (e.g. i am sure without py-deprecation the program doesn't work). i will test with each of them and correct the final portfile.
thank you very much again for your help and support.

Reply:
you could do `sudo port extract` and then check the `setup.py` in the "work" directory (```cd `port dir py-eyed3`/work```) to see what dependencies are actually needed. As I said before upstream has listed them incorrectly all under `install_requires` so that likely explains why `pip` installs them.